# ApplanixRaw Message

std_msgs/Header header

std_msgs/String applanixRaw



# applanixRaw @42 :Data;
